BursaryManagementAPI/Controllers/BBDAdminController.cs

```diff
@@ -0,0 +1,89 @@
+﻿using BusinessLogic;
+using BusinessLogic.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Data.SqlClient;
+using System;
+using System.Collections.Generic;
+
+namespace BursaryManagementAPI.Controllers
+{   
+    [Route("api/[controller]")]
+    [ApiController]
+    public class BBDAdminController : ControllerBase
+    {
+        private readonly BBDAdminBLL _BBDAdminBLL;
+        
+
+        public BBDAdminController(BBDAdminBLL BBDAdminBLL)
+        {            
+            _BBDAdminBLL = BBDAdminBLL;
+            
+        }
+
+        [HttpGet("GetAllUniversities")]
+        public ActionResult<IEnumerable<GetAllUniversities>> GetAllRequests()
+        {
+            try
+            {
+                var requests = _BBDAdminBLL.GetAllRequests();
+                return Ok(requests);
+            }
+            catch (Exception ex)
+            {
+                return StatusCode(500, $"Error retrieving universities: {ex.Message}");
+            }
+        }
+
+        [HttpPost("AddUniversity")]
+        public ActionResult Create([FromBody] AddUniversityAndUser newRequest)
+        {
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(ModelState);
+            }
+
+            try
+            {
+
+                _BBDAdminBLL.AddUniversity(newRequest);
+                return Ok("Student fund request created successfully!");
+            }
+            catch (Exception ex)
+            {
+                return StatusCode(StatusCodes.Status500InternalServerError, $"Error creating student fund request: {ex.Message}");
+            }
+        }
+
+        //[Authorize(Roles = Roles.BBDAdmin)]
+        [HttpPost("{applicationId}/approve")]
+        public ActionResult ApproveApplication(int applicationId)
+        {
+            try
+            {
+                _BBDAdminBLL.ApproveApplication(applicationId);
+                return Ok("Application approved successfully!");
```

Comment:
With this response, are able to get it as an object so we can access the value? I've been struggling with this one earlier.

Reply:
![image](https://github.com/GROUP2-Team-ANPV/UkukhulaBursary-Api/assets/62591258/81816152-8f45-4aa8-963e-5de04e893125)


Soething like this?

Reply:
Not sure how to achieve it on dotnet, I'll check